### PR TITLE
username_search now works when invoked from outside cloned dir

### PR DESCRIPTION
**Pull request by k4u5h1k**
Previously `username_checker` did not work when executed from another directory because the username_checker json path was appended to `os.cwd`.

This was fixed using
```
from os.path import dirname as up 
project_root = up(up(up(__file__)))
filepath = os.path.join(project_root, 
                       'data', 
                       'username_checker.json')
```

### CODE REVIEW

File: modules/osint/username_search.py [hunk 1]
```diff
@@ -58,15 +59,18 @@ def check(self, url, site, data):
 			OUTPUT['links'][site] = url
 
 def module_api(self):
-	query = self.options['query']
-	filepath = os.path.join(os.getcwd(), 'data', 'username_checker.json')
-	with open(filepath) as file:
-		data = json.loads(file.read())
-	thread(self, data, query,self.options['thread'])
-	output = OUTPUT
+        query = self.options['query']
```
Comment (saeeddhqan) [inline modules/osint/username_search.py]: Replace 4 spaces with \t.

**Review by saeeddhqan — COMMENTED** [review]
Check other modules. This bug may occur on the other modules as well.

File: modules/osint/username_search.py [hunk 2]
```diff
@@ -58,15 +59,18 @@ def check(self, url, site, data):
...
+        project_root = up(up(up(__file__)))
+        filepath = os.path.join(project_root, 
+                                'data', 
+                                'username_checker.json')
+        with open(filepath) as file:
+                data = json.loads(file.read())
```
Comment (z3y50n) [inline modules/osint/username_search.py]: I think you can replace with `data = json.load(file)`

File: modules/osint/username_search.py (hunk 1 above)
Comment (k4u5h1k) [reply modules/osint/username_search.py]: yes my bad vim screwed up my tabs
closing this pr and opening another one

File: modules/osint/username_search.py (hunk 2 above)
Comment (k4u5h1k) [reply modules/osint/username_search.py]: will do

**Pull request closed without merging**